src/eureka/S3_data_reduction/optspex.py

```diff
@@ -690,8 +695,7 @@ def optimize_wrapper(data, meta, log, apdata, apmask, apbg, apv0, gain=1,
     # Mask out NaNs and Infs
     optspec_ma = np.ma.masked_invalid(data.optspec.values)
     opterr_ma = np.ma.masked_invalid(data.opterr.values)
-    optmask = np.logical_or(np.ma.getmaskarray(optspec_ma),
-                            np.ma.getmaskarray(opterr_ma))
+    optmask = np.ma.getmaskarray(optspec_ma) + np.ma.getmaskarray(opterr_ma)
```

Comment:
`np.logical_or`, `np.ma.mask_or`, `|`, and `+` should all work equally well here, and all of them require the `np.ma.getmaskarray` to get the masks since `optspec_ma.mask` and `opterr_ma.mask` can sometimes be just `False` instead of an array of `False`s. Don't really remember why I switched this to use `+` - probably just for brevity, but I can use whatever function/symbol you prefer

Reply:
No preference. We can keep it as is.